tests/wamr-test-suites/test_wamr.sh

```diff
@@ -14,7 +14,9 @@ function help()
 {
     echo "test_wamr.sh [options]"
     echo "-c clean previous test results, not start test"
-    echo "-s {suite_name} test only one suite (spec|wasi_certification|wamr_compiler)"
+    echo "-s {suite_name} test only one suite (spec|standalone|sightglass|unit"
+    echo "                                     |malformed|wasi|wasi_certification"
```

Comment:
better only add the test suite that is available, for example, unit and standalone are unavailable, better not add them. I think maybe keep the original choice is good enough